alert-common/src/main/java/com/synopsys/integration/alert/common/rest/model/AlertPagedDetails.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * alert-common
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.alert.common.rest.model;
+
+import java.util.List;
+
+import com.synopsys.integration.alert.common.rest.api.ReadPageController;
+import com.synopsys.integration.util.Stringable;
+
+public class AlertPagedDetails<M extends Stringable> {
+    public static final Integer DEFAULT_PAGE_NUMBER = Integer.valueOf(ReadPageController.DEFAULT_PAGE_NUMBER);
+    public static final Integer DEFAULT_PAGE_SIZE = Integer.valueOf(ReadPageController.DEFAULT_PAGE_SIZE);
```

Comment:
I think I prefer keeping these in `AlertPagedModel` because that is the model we use in our API, but I could be convinced otherwise.